Formula/pyside@2.rb

```diff
@@ -38,20 +39,26 @@ class PysideAT2 < Formula
   end
 
   def install
-    # upstream issue: https://bugreports.qt.io/browse/PYSIDE-1684
-    unless OS.mac?
+    rpaths = if OS.mac?
+      site_packages = Language::Python.site_packages("python3")
+      prefix_relative_path = prefix.relative_path_from(prefix/site_packages/"PySide2")
+      [rpath, "@loader_path/#{prefix_relative_path}/lib"]
+    else
+      # Add missing include dirs on Linux.
+      # upstream issue: https://bugreports.qt.io/browse/PYSIDE-1684
       extra_include_dirs = [Formula["mesa"].opt_include, Formula["libxcb"].opt_include]
-
       inreplace "sources/pyside2/cmake/Macros/PySideModules.cmake",
                 "--include-paths=${shiboken_include_dirs}",
                 "--include-paths=${shiboken_include_dirs}:#{extra_include_dirs.join(":")}"
+      # Add rpath to qt@5 because it is keg-only.
+      [lib, Formula["qt@5"].opt_lib]
```

Comment:
Now that I think about it -- doesn't `brew` handle this already?

Reply:
Maybe there is a shim bypass happening inside of `pyside`?  Unfortunately the design of `pyside` makes it extremely difficult (for me at least) to actually figure out what it's doing when building C++ code so it's hard for me to say what's really happening.